pkg/assembler/backends/inmem/certifyBad.go

```diff
@@ -18,6 +18,7 @@ package inmem
 import (
 	"context"
 	"strconv"
+	"time"
```

Comment:
just a quick note that we will have to update the other backends to handle the new `timestamp` fields. This can be a follow on PR but we have to change those for the issue to close.

Reply:
What do you mean by other backends?

Reply:
[arango](https://github.com/guacsec/guac/tree/main/pkg/assembler/backends/arangodb) and [ent](https://github.com/guacsec/guac/tree/main/pkg/assembler/backends/ent) (eventually neptune but for now it's not functional). Which we can help you with.

Reply:
Ok, thanks! Now I understand. I think I will do this in a follow-up PR.